File: app/src/main/java/vn/loitp/app/activity/customviews/menu/drawerbehavior/DrawerBehaviorMainActivity.java
package vn.loitp.app.activity.customviews.menu.drawerbehavior;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import vn.loitp.app.R;
import vn.loitp.app.activity.customviews.menu.drawerbehavior.drawer.Advance3DDrawer1Activity;
import vn.loitp.app.activity.customviews.menu.drawerbehavior.drawer.AdvanceDrawer1Activity;
import vn.loitp.app.activity.customviews.menu.drawerbehavior.drawer.AdvanceDrawer2Activity;
import vn.loitp.app.activity.customviews.menu.drawerbehavior.drawer.AdvanceDrawer3Activity;
import vn.loitp.app.activity.customviews.menu.drawerbehavior.drawer.AdvanceDrawer4Activity;
import vn.loitp.app.activity.customviews.menu.drawerbehavior.drawer.AdvanceDrawer5Activity;
import vn.loitp.app.activity.customviews.menu.drawerbehavior.drawer.AdvanceDrawer6Activity;
import vn.loitp.app.activity.customviews.menu.drawerbehavior.drawer.DefaultDrawerActivity;

public class DrawerBehaviorMainActivity extends AppCompatActivity implements View.OnClickListener{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_drawer_behavior_main);

        findViewById(R.id.button_default).setOnClickListener(this);
        findViewById(R.id.button_advance1).setOnClickListener(this);
        findViewById(R.id.button_advance2).setOnClickListener(this);
        findViewById(R.id.button_advance3).setOnClickListener(this);
        findViewById(R.id.button_advance4).setOnClickListener(this);
        findViewById(R.id.button_advance5).setOnClickListener(this);
        findViewById(R.id.button_advance6).setOnClickListener(this);
        findViewById(R.id.button_advance_3d_1).setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()){
            case R.id.button_default:
                startActivity(new Intent(this, DefaultDrawerActivity.class));
                break;
            case R.id.button_advance1:
                startActivity(new Intent(this, AdvanceDrawer1Activity.class));
                break;
            case R.id.button_advance2:
                startActivity(new Intent(this, AdvanceDrawer2Activity.class));
                break;
            case R.id.button_advance3:
                startActivity(new Intent(this, AdvanceDrawer3Activity.class));
                break;
            case R.id.button_advance4:
                startActivity(new Intent(this, AdvanceDrawer4Activity.class));
                break;
            case R.id.button_advance5:
                startActivity(new Intent(this, AdvanceDrawer5Activity.class));
                break;

            case R.id.button_advance6:
                startActivity(new Intent(this, AdvanceDrawer6Activity.class));
                break;
            case R.id.button_advance_3d_1:
                startActivity(new Intent(this, Advance3DDrawer1Activity.class));
                break;
        }
    }
}

File: app/src/main/java/vn/loitp/app/activity/customviews/menu/drawerbehavior/DrawerBehaviorMainActivity.kt
package vn.loitp.app.activity.customviews.menu.drawerbehavior

import android.content.Intent
import android.os.Bundle
import android.view.View
import com.annotation.IsFullScreen
import com.annotation.LayoutId
import com.annotation.LogTag
import com.core.base.BaseFontActivity
import com.core.utilities.LActivityUtil
import kotlinx.android.synthetic.main.activity_drawer_behavior_main.*
import vn.loitp.app.R
import vn.loitp.app.activity.customviews.menu.drawerbehavior.drawer.*

@LayoutId(R.layout.activity_drawer_behavior_main)
@LogTag("DrawerBehaviorMainActivity")
@IsFullScreen(false)
class DrawerBehaviorMainActivity : BaseFontActivity(), View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        buttonDefault.setOnClickListener(this)
        buttonAdvance1.setOnClickListener(this)
        buttonAdvance2.setOnClickListener(this)
        buttonAdvance3.setOnClickListener(this)
        buttonAdvance4.setOnClickListener(this)
        buttonAdvance5.setOnClickListener(this)
        buttonAdvance6.setOnClickListener(this)
        buttonAdvance3d_1.setOnClickListener(this)
    }

    override fun onClick(v: View) {
        when (v) {
            buttonDefault -> startActivity(Intent(this, DefaultDrawerActivity::class.java))
            buttonAdvance1 -> startActivity(Intent(this, AdvanceDrawer1Activity::class.java))
            buttonAdvance2 -> startActivity(Intent(this, AdvanceDrawer2Activity::class.java))
            buttonAdvance3 -> startActivity(Intent(this, AdvanceDrawer3Activity::class.java))
            buttonAdvance4 -> startActivity(Intent(this, AdvanceDrawer4Activity::class.java))
            buttonAdvance5 -> startActivity(Intent(this, AdvanceDrawer5Activity::class.java))
            buttonAdvance6 -> startActivity(Intent(this, AdvanceDrawer6Activity::class.java))
            buttonAdvance3d_1 -> startActivity(Intent(this, Advance3DDrawer1Activity::class.java))
        }
        LActivityUtil.tranIn(this)
    }
}